.circleci/config.yml

```diff
@@ -47,6 +77,87 @@ jobs:
       - run:
           name: Run Checks
           command: bin/ci-checks-js.sh
+  android-device-checks:
+    docker:
+    - image: circleci/android:api-27-node8-alpha
+    steps:
+      - checkout
+      - run:
+          name: Checkout Gutenberg
+          command: |
+            git submodule init
+            git submodule update --recursive
```

Comment:
Can we extract this into a command since its repeated a few times? Something like this would work:

```
full-checkout:
  steps:
    - checkout
    - run:
        name: Checkout Gutenberg
        command: |
          git submodule init
          git submodule update --recursive
```

Reply:
Using `git submodule update --init --recursive` directly should be fine too, right?

Reply:
Yes, that would reduce the two lines to one, but I think its still useful to extract so that we don't repeat it in the config 3 times.